site/en/reference/cli/concepts/udp.md

```diff
@@ -0,0 +1,185 @@
+# Send UDP Messages with OT CLI
```

Comment:
Should we update to be more clear how the CLI is intended for use, something like:

`# Test UDP functionality with OT CLI`

Reply:
I like it; will change to that.  @abtink - agree?